### PR TITLE
[v6r15] Various minor fixes

### PR DESCRIPTION
**Pull request by fstagni**
Replaces https://github.com/DIRACGrid/DIRAC/pull/3028 thwat was pointing to integration

### CODE REVIEW

**Comment by fstagni** [comment]
Review, please

File: Core/Utilities/MySQL.py [hunk 1]
```diff
@@ -1488,7 +1477,7 @@ def insertFields( self, tableName, inFields = None, inValues = None, conn = None
                          ( table, inFieldString, inValueString ), conn, debug = True )
 
 
-  def executeStoredProcedure( self, packageName, parameters, outputIds, output = True, array = None, conn = False ):
+  def executeStoredProcedure( self, packageName, parameters, outputIds ):
```
Comment (chaen) [inline Core/Utilities/MySQL.py]: Not sure of that one. I put this useless params to have the same interface as the Oracle interface

File: RequestManagementSystem/ConfigTemplate.cfg [hunk 2]
```diff
@@ -10,17 +10,17 @@ Services
   }
   ReqProxy
   {
-    Port = 9198
+    Port = 9161
```
Comment (iueda) [inline RequestManagementSystem/ConfigTemplate.cfg]: Why the change of the port?

Comment (fstagni) [inline RequestManagementSystem/ConfigTemplate.cfg]: Well spotted. This change of port is a bugfix: I have been asked to change the default port for ReqProxy to 9161 as 9198 was already taken.

Comment (fstagni) [inline RequestManagementSystem/ConfigTemplate.cfg]: I reverted to 9198 in light of the discussions of the forum

File: Core/Utilities/MySQL.py (hunk 1 above)
Comment (fstagni) [inline Core/Utilities/MySQL.py]: There is, as of today, no connections between the MySQL class and the OracleDB class, so I think this is not an issue. 
We can, on a side note, discuss that this is not ideal, but it won't affect this change.

Comment (chaen) [inline Core/Utilities/MySQL.py]: I agree with you. We just need to be now concious that the interfaces are not compatible, which I think they were up to now, but I don't think we really care, especially with the SQLAlchemy direction

File: RequestManagementSystem/ConfigTemplate.cfg (hunk 2 above)
Comment (rajanandakumar) [inline RequestManagementSystem/ConfigTemplate.cfg]: Hi, I requested the change to 9161, as that is the port used by LHCb (and I thought everybody) for this. New installations were failing and each installation needed this service to be deleted and re-installed by hand due to this bug.

What is the reason this was reverted?

Thanks,
Raja.

Comment (fstagni) [inline RequestManagementSystem/ConfigTemplate.cfg]: Please check the google forum for an explanation

Comment (rajanandakumar) [inline RequestManagementSystem/ConfigTemplate.cfg]: Hi Federico,

I have so far found no explanation in the google forum. Only as question from Ueda and your response stating that the change has been reverted (but no explanation).

Cheers,
Raja.

File: Core/Utilities/MySQL.py [hunk 3]
```diff
@@ -458,7 +458,7 @@ def __escapeString( self, myString ):
     It also includes quotation marks " around the given string
     """
 
-    retDict = self.__getConnection()
+    retDict = self._getConnection()
```
Comment (zmathe) [inline Core/Utilities/MySQL.py]: Why Do you changed this method from private to protected? In low level private methods are very useful.

File: RequestManagementSystem/ConfigTemplate.cfg (hunk 2 above)
Comment (fstagni) [inline RequestManagementSystem/ConfigTemplate.cfg]: We should be careful with changing the port numbers as some other VOs 
that are not LHCb may have already assigned 9161 to some other service.

File: Core/Utilities/MySQL.py (hunk 3 above)
Comment (fstagni) [inline Core/Utilities/MySQL.py]: This method already existed, and was just calling the private method. I simplified the code.

File: TransformationSystem/Client/FileReport.py [hunk 4]
```diff
@@ -1,27 +1,33 @@
...
-from DIRAC import S_OK, S_ERROR
+import copy
+
+from DIRAC import S_OK
 from DIRAC.Core.Utilities                                     import DEncode
 from DIRAC.TransformationSystem.Client.TransformationClient   import TransformationClient
 from DIRAC.RequestManagementSystem.Client.Operation           import Operation
 
-import copy
-
```
Comment (zmathe) [inline TransformationSystem/Client/FileReport.py]: if we follow PEP8 import always at the end for example:
from a import b
from a import c

import a

File: WorkloadManagementSystem/Executor/InputData.py [hunk 5]
```diff
@@ -313,8 +313,10 @@ def __getSiteCandidates( self, okReplicas, vo ):
       siteSet = set()
       for seName in replicas:
         result = self.__getSitesForSE( seName )
-        if result['OK']:
-          siteSet.update( result['Value'] )
+        self.jobLog.warn( "Could not get sites for SE %s: %s" % ( seName, result[ 'Message' ] ) )
```
Comment (zmathe) [inline WorkloadManagementSystem/Executor/InputData.py]: something wrong here. I do not see if and if result['OK'], there is no result['Message']

File: WorkloadManagementSystem/Executor/InputData.py [hunk 6]
```diff
@@ -313,8 +313,10 @@ def __getSiteCandidates( self, okReplicas, vo ):
...
+        if not result['OK']:
```
Comment (zmathe) [inline WorkloadManagementSystem/Executor/InputData.py]: ah self.jobLog.warn line has to go here

File: tests/Jenkins/dirac-proxy-download.py [hunk 7]
```diff
@@ -21,10 +19,11 @@
 args = Script.getPositionalArgs()
 dn = ' '.join( args )
 
-import os
 uid = os.getuid()
 from DIRAC.FrameworkSystem.Client.ProxyManagerClient        import gProxyManager
 
+print "Getting proxy for User DN: %s, User role %s" % (dn, role)
```
Comment (zmathe) [inline tests/Jenkins/dirac-proxy-download.py]: gLogger.notice?

File: Core/Utilities/MySQL.py (hunk 3 above)
Comment (zmathe) [inline Core/Utilities/MySQL.py]: Yes. It is simplified but it is complicated the debug. We know all the services are using _getConnection() method. MySQL is using __getConnection internally. So If we want to debug the mysql db, we can not do it easily as now we have calls from services..

File: tests/Jenkins/dirac-proxy-download.py (hunk 7 above)
Comment (fstagni) [inline tests/Jenkins/dirac-proxy-download.py]: nah, this is used only for jenkins tests

File: WorkloadManagementSystem/Executor/InputData.py (hunk 5 above)
Comment (fstagni) [inline WorkloadManagementSystem/Executor/InputData.py]: Thanks, corrected
